bank_tiles.h

```diff
@@ -0,0 +1,52 @@
+/*
+ *  yosys -- Yosys Open SYnthesis Suite
+ *
+ *  Copyright (C) 2012  Clifford Wolf <clifford@clifford.at>
+ *  Copyright (C) 2019  The Symbiflow Authors
+ *
+ *  Permission to use, copy, modify, and/or distribute this software for any
+ *  purpose with or without fee is hereby granted, provided that the above
+ *  copyright notice and this permission notice appear in all copies.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ *  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ *  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ *  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ *  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ *  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ *  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *
+ */
+#include "kernel/log.h"
+#include "json.hpp"
+
+#define PART_JSON "PRJXRAY_PART_JSON"
+
+USING_YOSYS_NAMESPACE
+// Coordinates of HCLK_IOI tiles associated with a specified bank
+using BankTilesMap = std::unordered_map<int, std::string>;
+
+// Find the part's JSON file with information including the IO Banks
```

Comment:
What json file are you using here?

Reply:
I plan to use the json that we have in prjxray's db for a given part, for example `xc7a35tcsg324-1.json` with an extra entry that we can generate in one of the fuzzers:
<pre>
    "iobanks": {
        "14" : "X1Y26",
        "15" : "X1Y78",
        "16" : "X1Y130",
        "34" : "X113Y26",
        "35" : "X113Y78"
    }
</pre>